articles/extensions/authorization-extension/v2/implementation/setup.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Setting Up the Authorization Extension
+description: How to set up the Authorization Extension
+toc: true
+---
+
+# Authorization Extension: Setup
+
+Once you've configured the extension, you can manage your users, as well as the groups, roles and permissions for your users in the Authorization Extension dashboard.
+
+## Entities Available for Management
+
+In this paragraph, we'll briefly cover the basics of users, groups, roles, and permissions.
+
+Let's say that you have an application that is accessible to everyone within your corporation. The **users** are the individuals to whom you'd like to grant access to your application.
+
+If you have a large number of users, managing the access rights and permissions each one individually can become unwieldy. The **groups** functionality helps make this process easier. For example, you might have your groups reflect the various departments of your organization: accounting, information technology, engineering, support, and so on. You might also create nested groups, such as dividing the engineering group into two nested groups: internal tools and client-facing applications. Your organization hierarchy therefore looks like this:
+
+* Corporation
```

Comment:
A diagram would work so much better here. Can you address this in a separate PR? (since getting the diagram from the design team will take some time)